### DISCUSSION TITLE
BDD Cucumber with c++ on Windows

### DISCUSSION THREAD

**Discussion by Austeja-Sut** [cucumber/common#2216] [Q&A C++]
Hey! 

Can someone explain how to possibly run BDD Cucumber method with c++ coding language on compiler such as Clion or CodeBlocks?

I tried based on Chat GPT, but always run into some errors I don't understand.

**Comment by luke-hill** [comment]
ping @ursfassler was dealing with improving the cucumber cpp repository

**Reply by ursfassler** [reply]
Hi @Austeja-Sut,
cucumber-cpp is built on Windows in the CI on github. This also works as a guide to setup it yourself. The relevant files are:
https://github.com/cucumber/cucumber-cpp/blob/main/.github/workflows/windows-build.yml
https://github.com/cucumber/cucumber-cpp/blob/main/run-windows.ps1

I also wrote a blog article, but for Linux: https://iqilio.ch/blog/2024/cucumber-cpp-first-steps/

Hope this helps!
